### PR TITLE
upgrade dependencies to prepare for Legal approvals

### PR DESCRIPTION
**Pull request by russgold**
Upgraded to the latest version of all dependencies.

### CODE REVIEW

**Comment by russgold** [comment]
Searching the database of licensed dependencies, I found that the latest approved versions are not necessarily the ones I have here.

I cannot find any currently approved versions of commons-fileupload, commons-io, commons-logging or fluent-hc, so we'll need to do those from the start, and we might as well use the latest versions.

**Review by rjeberhard — APPROVED** [review]

**Comment by rjeberhard** [comment]
Approving, but of course, we need to have the Legal approvals ready before merging.

File: pom.xml
Comment (alai8) [inline pom.xml:84]: is this a new dependency?

Comment (russgold) [reply pom.xml:84]: It was being pulled in implicitly, and I made it explicit because we are requesting authorization to use a specific version.

**Review by alai8 — APPROVED** [review]
The change looks good, although I do not know what the versions that we should be using for these dependencies.

**Review by mriccell — APPROVED** [review]

**Review by rjeberhard — APPROVED** [review]

**Pull request merged**
